functions/src/download-readings.ts

```diff
@@ -140,8 +140,8 @@ function generateAverageReadingsCsv(): void {
 
   // Current-reading collection has single doc
   const currentReadingDocRef = firestore
-    .collection('/current-reading')
-    .doc('pm25');
+    .collection('current-reading')
```

Comment:
Does this correspond to some other change? I can't remember what this was about.

Reply:
Just for consistency across the codebase